docs/src/plasma/plasma-cnc-primer.adoc

```diff
@@ -519,11 +519,11 @@ Because the full scale is 10 Volts, then the frequency per Volt is:
 
 So assuming we have a 5 Volt input, the calculated frequency would be:
 
-(2520 Hz/V * 5 V) + 3,800 Hz = 16,400 Hz
+(2520 Hz/V * 5 V) + 3800 Hz = 16,400 Hz
 
 So now it should be fairly clear how to convert the frequency to its voltage equivalent:
 
-Volts = (frequency (Hz) - 3,800 Hz) / 2,520 Hz/V
+Volts = (frequency [Hz] - 3,800 Hz) / 2,520 Hz/V
```

Comment:
This I meant with "next occurrence that is not displayed here"
We should have it consistent.

```suggestion
Volts = (frequency [Hz] - 3800 Hz) / 2520 Hz/V
```